src/cloud/architecture/starter-architecture.md

```diff
@@ -0,0 +1,135 @@
+
+-  Fastly for HTTP caching and CDN
+-  Nginx web server speaking to PHP-FPM, one instance with multiple workers
+-  Redis server
+-  Elasticsearch for searching for {{site.data.var.ece}} 2.2 and later
+
+### Services {#cloud-arch-services}
+
+{{site.data.var.ece}} currently supports the following services: PHP, MySQL (MariaDB), Elasticsearch (Magento 2.2.x and later), Redis, and RabbitMQ.
+
+Each service runs in its own secure container. Containers are managed together in the project. Some services are built-in, such as the following:
+
+-  HTTP router (handling incoming requests, but also caching and redirects)
+-  PHP application server
+-  Git
+-  Secure Shell (SSH)
+
+You can even have multiple applications running in the same project. Building a microservice oriented architecture with {{site.data.var.ee}} is as easy as managing a monolithic application.
+
+### Software versions {#cloud-arch-software}
+
+{{site.data.var.ece}} uses the Debian GNU/Linux operating system and the [NGINX](https://glossary.magento.com/nginx) web server. You cannot upgrade this software, but you can configure versions for the following:
+
+-  [PHP][]
+-  [MySQL][]
+-  [Redis][]
+-  [RabbitMQ][]
+-  [Elasticsearch][]
+
+In the Staging and Production environments, you use Fastly for CDN and caching. When your environment is initially provisioned, we install the latest version of the Fastly CDN extension for Magento. You can upgrade the extension to get the latest bug fixes and improvements. See [Fastly CDN module for Magento 2][]. You also have access to [New Relic][] for performance management.
+
+You use the following files to configure the software versions that you want to use in your implementation.
+
+-  [`.magento.app.yaml`][magento.app.yaml]
+-  [`routes.yaml`][routes.yaml]
+-  [`services.yaml`][services.yaml]
+
+### Backup and disaster recovery {#backup}
+
+You can create a snapshot of your database and file system using the Project Web Interface or the CLI. The snapshot includes your deployed code, installed software and services, and data. See [Snapshots and backup management][].
+
+## Prepare for development {#develop}
+
+The following workflow summarizes the process to branch your code, develop, and deploy your store:
+
+-  Set up your local environment
+-  Clone the `master` branch from the Project to your local environment
+-  Create a `staging` branch from `master`
+-  Create branches for development from `staging`
+-  Push code to Git that builds and deploys to an environment for testing
+
+See the following sections for detailed instructions and walk-throughs to develop, test, and deploy your store:
+
+-  [Starter develop and deploy workflow][]
+-  [Docker development][] (local development environment enabled by {{site.data.var.mcd-prod}})
+-  [Manage branches][]
+-  [Deploy your store][]
+-  [Site launch][]
+
+<!--Link definitions-->
+[Deploy your store]: {{ site.baseurl }}/cloud/live/stage-prod-live.html
```

Comment:
Long definition lists like this are easier to manage if you alphabetize. Just a thought.